Makefile

```diff
@@ -27,8 +27,12 @@ S3_BUCKET ?= crossplane.releases/oam
 HELM_BASE_URL = https://charts.crossplane.io
 HELM_S3_BUCKET = crossplane.charts
 HELM_CHARTS_DIR=$(ROOT_DIR)/charts
-HELM_CHARTS = oam-kubernetes-runtime
+LEGACY_HELM_CHARTS = oam-kubernetes-runtime-legacy
+HELM_CHARTS = oam-kubernetes-runtime $(LEGACY_HELM_CHARTS)
```

Comment:
```suggestion
LEGACY_HELM_CHARTS = oam-kubernetes-runtime-legacy
HELM_CHARTS = oam-kubernetes-runtime, $(LEGACY_HELM_CHARTS)
```

Reply:
Note that I didn't check that these are comma separated so I could be wrong there - space separated might be correct.

Reply:
I ran `make -j build.all` and it showed using space is the correct way.

If using comma, it will hit an issue as `oam-kubernetes-runtime,` is regarded as the first item of the list.
```
find: /Users/zhouzhengxi/Programming/golang/src/github.com/crossplane/oam-kubernetes-runtime/charts/oam-kubernetes-runtime,: No such file or directory
find: /Users/zhouzhengxi/Programming/golang/src/github.com/crossplane/oam-kubernetes-runtime/charts/oam-kubernetes-runtime,: No such file or directory
cp: cannot stat '/Users/zhouzhengxi/Programming/golang/src/github.com/crossplane/oam-kubernetes-runtime/charts/oam-kubernetes-runtime,/values.yaml.tmpl': No such file or directory
make[1]: *** [helm.prepare.oam-kubernetes-runtime,] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [build.all] Error 2
```